### ISSUE TITLE
Test using Lighthouse

### ISSUE THREAD

**Issue by anthkris** [anthkris/mws-restaurant-stage-1#26]
Follow the recommendations provided by Lighthouse to achieve the required performance targets: 

- Progressive Web App score should be at 90 or better.
- Performance score should be at 70 or better.
- Accessibility score should be at 90 or better.

You can audit your site's performance with Lighthouse by using the Audit tab of Chrome Dev Tools.

**Comment by anthkris** [comment]
<img width="860" alt="captura de pantalla 2018-07-07 a la s 2 31 50 p m" src="https://user-images.githubusercontent.com/10858542/42414032-9dc6f326-81f2-11e8-94b4-4e9bfba351b1.png">

Good lighthouse scores.
